main.tf

```diff
@@ -24,6 +24,8 @@ locals {
     "Reader",
     "Security Reader"
```

Comment:
@livnoni Please leave only the `Reader`, as the `Security Reader` is a sub-group of it (my bad)

Reply:
got it, removed _Security Reader_